### DISCUSSION TITLE
Showing About Me in Search results

### DISCUSSION THREAD

**Discussion by mschillinghaas** [microsoft-search/pnp-modern-search#2683] [Q&A]
Im trying to get the Delve About Me data to show in a Users search results. I can search by keywords that exist in About me so i know its looking but i cant get the About Me data to show under the users results. Name, Title, Email and Thumbnail are all returned but no About me. Ive added About Me to the list of Selected Properties and added that Property to the manage People Fields {{slot item @root.slots.AboutMe}}.

**Comment by gretchunkim** [comment]
@mschillinghaas What data source are you using? Is it SharePoint or Graph? If Graph, are you able to retrieve your target response from Graph Explore or any other tool? This will be the first place to check to troubleshoot. Later, if  no response, then you'll need to identify if it's the field property configuration that's causing the issue (retrieve, query, search, etc). 

Here's a get HTTP you can use to test: 
https://graph.microsoft.com/v1.0/me/aboutMe

**Comment by wobba** [comment]
You need to type `AboutMe` and hit enter in the selected properties field to return it. Now you can reference the manged property in a slot or directly.

![image](https://user-images.githubusercontent.com/51104/212722741-7e06a2fe-0942-4573-a810-443a6a6db52c.png)
